src/metabase/api/card.clj

```diff
@@ -125,6 +128,21 @@
   [_filter-option model-id]
   (cards-for-segment-or-metric :segment model-id))
 
+(defmethod cards-for-filter-option* :stale
+  [_]
+  (let [card-id->field-usages (group-by :card_id
+                                        (t2/select :model/FieldUsage
+                                                   {:select [:fu.* [:f.name :new_column_name] [:t.name :new_table_name]]
```

Comment:
1. what's `new_column_name`, `new_table_name` is for?
2. We already joined with Field, Table here, so `metabase_table.name, metabase_field.name, metabase_field.active` can be selected from those tables, I don't see any reason we need to keep track of those in `field_usage`